ADA/ada.qmd

```diff
@@ -48,7 +57,13 @@ The ADA team also intends to establish an analyst Community of Practice that:
 
 * Identifies future opportunities to support the Department's Strategic Data Transformation 
 
-A list of current ADA champions can be found on the [ADA intranet page](https://educationgovuk.sharepoint.com/sites/lvewp00505), they are able to provide advice and support if you run into any issues. If you wish to become an ADA champion then please contact the [ADA team](mailto:ADAPTTeam@education.gov.uk).
+You can access support in the following places:
+
+- The [ADA user group](https://teams.microsoft.com/l/team/19%3AGQW24K3V6M873QzUwpBIuyf8KZCxYByRawVsKQvpe-81%40thread.tacv2/conversations?groupId=c70bb59d-c5df-4af5-849f-36196ed14ba3&tenantId=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9) on Microsoft Teams 
```

Comment:
Nitpicky: inconsistent full stops on the end of bullet points are back

Reply:
I'm sure a gremlin is adding these back in

Reply:
Fixed though